content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Add a new Custom Ingress Gateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/setup/).
+
+## Configuring the custom ingress gateway
+
+1. Check if [cert-manager](https://github.com/helm/charts/tree/master/stable/cert-manager) was installed using Helm with the following command:
+
+    {{< text bash >}}
+
+    $ helm ls
+
+    {{< /text>}}
+
+    The output should be similar to the example below and show cert-manager with a `STATUS` of `DEPLOYED`:
+
+    {{< text plaintext >}}
+
+    NAME   REVISION UPDATED                  STATUS   CHART                     APP VERSION   NAMESPACE
+    istio     1     Thu Oct 11 13:34:24 2018 DEPLOYED istio-1.0.2               1.0.2         istio-system
+    cert      1     Wed Oct 24 14:08:36 2018 DEPLOYED cert-manager-v0.6.0-dev.2 v0.6.0-dev.2  istio-system
+
+    {{< /text >}}
+
+1. To create the cluster's issuer, apply the following configuration:
+
+    {{< warning_icon >}} Change the cluster's issuer provider with your own configuration values. The example uses the values under `route53`.
+
+    {{< text yaml >}}
+
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: ClusterIssuer
+    metadata:
+      name: letsencrypt-demo
+      namespace: kube-system
+    spec:
+      acme:
+        # The ACME server URL
+        server: https://acme-v02.api.letsencrypt.org/directory
+        # Email address used for ACME registration
+        email: <REDACTED>
+        # Name of a secret used to store the ACME account private key
+        privateKeySecretRef:
+          name: letsencrypt-demo
+        dns01:
+          # Here we define a list of DNS-01 providers that can solve DNS challenges
+          providers:
+          - name: aws-dns
+            route53:
+              accessKeyID: <REDACTED>
+              region: eu-central-1
+              secretAccessKeySecretRef:
+                name: prod-route53-credentials-secret
+                key: secret-access-key
+
+    {{< /text >}}
+
+1. If you use the `route53` provider, you must provide a secret to perform DNS ACME Validation. To create the secret, apply the following configuration file:
```

Comment:
I'm not familiar with AWS , route53, or cert-manager, so have no idea what I need to do to on a different cloud platform.

Reply:
Updated with a link to cert-manager provider configuration